src/content/smart-bulletins/029.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Smart Bulletin No. 029"
+intro: "Interim Federal Acquisition Regulation (FAR) Rule Prohibiting Acquiring Certain Telecommunications and Video Surveillance Equipment or Services"
+number: "No. 029"
+order: 029
+businessline: "Purchase and Integrated (when used as a purchase card)"
+published: "Aug 13, 2019"
+lastupdated: "Jun 2, 2020"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| August 13, 2019| Implementation of Smart Bulletin #29 |
+| June 2, 2020 | Updated FAQ |
+
+## Effective Date
+
+August 13, 2019
+
+## Business Line(s) Affected
+
+Purchase and Integrated (when used as a purchase card)
+
+## Introduction
+
+The Department of Defense, General Services Administration, and the National Aeronautics and Space Administration issued an interim rule amending the FAR to implement section 889(a)(1)(A) of the John S. McCain National Defense Authorization Act for Fiscal Year 2019 (Public Law 115-232).
+
+Effective August 13, 2019, the interim rule revises the FAR and prohibits agencies from procuring, obtaining, extending, or renewing a contract (including micro-purchases) for equipment, systems, or services that use covered telecommunications and/or video surveillance equipment or services. This includes items that are substantial or essential to systems or are critical to any system.
+
+## Summary
+
+1. The interim FAR rule adopts the definition of critical technologies included in the Foreign Investment Risk Review Modernization Act of 2018 and amends sections of the FAR designed to ensure the United States is protected from certain risks regarding foreign actors.
+2. The Federal Government is prohibited from acquiring any equipment, system, or service that uses covered telecommunications equipment or services as asubstantial or essential component of any system, or as critical technology aspart of any system. This prohibition on acquiring covered telecommunications applies to acquisitions at all dollar levels including micro-purchases, unless a waiver is granted or an exception applies.
+
+## Action
+
+Agency headquarters A/OPCs shall ensure all personnel involved in purchase card use and management are aware of this new statutory prohibition on acquiring covered telecommunications. The attached Frequently Asked Questions (FAQs) developed by the Center for Charge Card Management (CCCM) is provided to assist in this effort. As noted in the FAQs:
```

Comment:
Agency headquarters A/OPCs shall ensure all personnel involved in purchase card use and management are aware of this new statutory prohibition on acquiring covered telecommunications. The attached Frequently Asked Questions (FAQs) developed by the Center for Charge Card Management (CCCM) are provided to assist in this effort. As noted in the FAQs:
To assist cardholders and card managers, a list of the sources (“vendors”) of the covered telecommunications equipment and/or services is posted on the GSA SmartPay® website at https://smartpay.gsa.gov/content/prohibited-vendor-list.
GSA CCCM online purchase cardholder and A/OPC training was updated to reflect the prohibition on acquiring covered telecommunications.
Agency cardholder or card manager questions regarding the propriety of a particular telecommunications purchase should be directed to the appropriate agency acquisition policy staff. Other questions or comments regarding this Smart Bulletin should be directed to CCCM through email at: gsa_smartpay@gsa.gov.